experimental/dds/ot/sharejs/json1/package.json

```diff
@@ -36,9 +72,11 @@
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../../../.prettierignore",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "mocha --recursive dist/test -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha": "npm run test:mocha:cjs && npm run test:mocha:esm",
+		"test:mocha:cjs": "mocha  --recursive \"dist/test/*.spec.*js\" --exit --project src/test/tsconfig.cjs.json -r node_modules/@fluid-internal/mocha-test-setup",
```

Comment:
Do you have any idea about the `--project` mocha option. It is not documented at https://mochajs.org/#command-line-usage and our projects are special with the dual build. If this isn't doing anything, we should remove it. I also wonder about `-r` but don't have connection to config to worry about.

Reply:
@DLehenbauer , @sonalideshpandemsft ^^ don't want this lost in the conversions. We should understand mocha --project. Is it real?

Reply:
Oh, looks like #19774 aims to remove these. Nice :)